includes/admin.form.inc

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * @file
+ * Admin callbacks for Islandora Videojs
+ */
+
+/**
+ * Admin form.
+ */
+function islandora_videojs_admin($form, &$form_state) {
+  // Get settings.
+  $form = array();
+  $form['islandora_videojs_hls_library'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Videojs-contrib-hls library'),
+    '#description' => t('Include the videojs-contrib-hls library js, videojs.contrib-hls.js?'),
+    '#default_value' => variable_get('hls_library', FALSE),
+    '#element_validate' => array('videojs_hls_validate'),
+  );
+
+  return system_settings_form($form);
+}
+
+/**
+ * Validates the above element.
+ */
+function videojs_hls_validate($element, $form, &$form_state) {
```

Comment:
Can we prefix this with the form name, and use the form element name in the function, just for consistency sake with other modules etc. For example:

`islandora_videojs_admin_islandora_videojs_hls_library_validate `